src/main/java/com/google/sps/data/Post.java

```diff
@@ -98,6 +144,7 @@ public void requestToPost(HttpServletRequest request) {
     numberOfPeopleItFeeds = Integer.parseInt(numberOfPeopleItFeedsUnparsed);
     typeOfFood = typeOfFoodUnparsed;
     description = descriptionUnparsed;
+    rank = numberOfPeopleItFeeds + getDuration();
```

Comment:
This is a good start but I would recommend (possibly in a follow-up PR) normalizing both of these values e.g. if numberOfPeopleItFeeds is too large, it will kind of overshadow the contribution of getDuration(). 

Trigonometric functions (sin, cosine) and the logistic regression function can help you with this. Let me know if you want to chat more about this.

Reply:
Just set up a chat with you about it!